src/states_screens/credits.cpp

```diff
@@ -393,6 +394,10 @@ void CreditsScreen::eventCallback(GUIEngine::Widget* widget,
     {
         StateManager::get()->escapePressed();
     }
+    if (name == "donate")
+    {
+        Online::LinkHelper::OpenURL("https://SuperTuxKart.net/Donate"); // Open donation page
```

Comment:
This should preferably come from the stk_config.xml config file and not be hardcoded, I think? @hiker would you agree?

Reply:
This still applies